app/test_main.py

```diff
@@ -1,3 +1,35 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_array",
+    [
+        pytest.param(
+            0, 0, [0, 0],
+            id="should return [0, 0] for cat and dog with age 0"
+        ),
+        pytest.param(
+            14, 14, [0, 0],
+            id="should return [0, 0] for cat and dog with age < 15"
+        ),
+        pytest.param(
+            24, 23, [2, 1],
+            id="should return [2, 1] for cat and dog with age 24 and 23"
+        ),
```

Comment:
you also need to test 15, 23 and 24 years for both cats and dogs because these values are edge cases